src/test/java/net/openhft/chronicle/core/JvmTest.java

```diff
@@ -303,4 +307,25 @@ public void removingTag() {
         assertEquals("value", actual);
     }
 
+    @Test
```

Comment:
It would be easier to understand if this was based on a real examples such as Base64 rather than Foo, RealAnno and Anno alias. as this is harder to understand what is going on.